src/test/groovy/com/mateusz/jakuszko/roomforyou/service/ApartmentDbServiceSpecification.groovy

```diff
@@ -1,15 +1,17 @@
 package com.mateusz.jakuszko.roomforyou.service
 
+import com.mateusz.jakuszko.roomforyou.TestcontainersSpecification
 import com.mateusz.jakuszko.roomforyou.entity.Apartment
 import com.mateusz.jakuszko.roomforyou.entity.Reservation
 import com.mateusz.jakuszko.roomforyou.exceptions.NotFoundException
 import com.mateusz.jakuszko.roomforyou.repository.ApartmentRepository
 import org.springframework.beans.factory.annotation.Autowired
 import org.springframework.boot.test.context.SpringBootTest
-import spock.lang.Specification
+import org.springframework.transaction.annotation.Transactional
 
 @SpringBootTest
-class ApartmentDbServiceSpecification extends Specification {
+@Transactional
```

Comment:
Jaki był powód że dodałeś tutaj `@Transactional`?

Reply:
Wstyd się przyznać, leciał mi org.hibernate.LazyInitializationException, przy odczycie z bazy danych. Mam świadomość, że to brzydkie rozwiązanie problemu. Jesteś w stanie podpowiedzieć jak można w tym przypadku inaczej to rozwiązać. Wyjatek rzucała **def "should save and load apartment from db"**

Reply:
Ok, w takim razie póki co daje ✅  ale jak tylko będzie okazja porozmawiamy sobie dlaczego @Transactional w testach to średni pomysł i jak sobie z tym radzić 😉